apps/credential-service/BUILD.bazel

```diff
@@ -0,0 +1,29 @@
+load("//tools:defaults.bzl", "esbuild_esm_bundle")
+load("@build_bazel_rules_nodejs//:index.bzl", "copy_to_bin", "nodejs_binary")
+
+nodejs_binary(
+    name = "serve",
+    data = [
+        ":bin_files",
+    ],
+    entry_point = ":credential-service.js",
+)
+
+copy_to_bin(
+    name = "bin_files",
+    srcs = [
+        "package.json",
+    ],
+)
+
+esbuild_esm_bundle(
+    name = "credential-service",
```

Comment:
ooc: how do you intend to deploy the script?

Reply:
My plan is to handle it in a follow up, the plan is to run the service in a container.  I was looking at `rules_docker` but its set to be replaced so I am still sorting out specifics.

Reply:
Sounds good. was mostly asking out of curiosity given the checked in script